src/main/java/com/gmail/inverseconduit/bot/DefaultBot.java

```diff
@@ -58,7 +61,10 @@ private void processMessage(final ChatMessage chatMessage) {
         final String trigger = AppContext.INSTANCE.get(BotConfig.class).getTrigger();
         if ( !chatMessage.getMessage().startsWith(trigger)) { return; }
 
-        commands.stream().filter(c -> chatMessage.getMessage().replace(trigger, "").startsWith(c.getName())).findFirst().map(c -> c.execute(chatMessage))
+        final String commandName = chatMessage.getMessage().replace(trigger, "").split(" ")[0];
```

Comment:
@mangstadt Yeah but you're still using `.replaceAll`, it's that bit simpler to use `.replaceFirst`, as it tells the regex engine to just replace first and not to keep trying (which always fails for the second attempt).